src/chrome/content/rules/openembedded.org.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="openembedded.org">
+	<target host="openembedded.org" />
+	<target host="www.openembedded.org" />
+	<target host="git.openembedded.org" />
```

Comment:
Please sort the target domains in alphabetical orders, per https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering. Otherwise, lgtm.

Reply:
Done.